canonical/store-components
# make Libraries and platform icons render for interface cards

## Done
-made Libraries and platform icons render for interface cards

## QA
-go to https://store-components-57.demos.haus/?path=/story/interfacecard--default-story and see if there are "Libraries" on the left and platform icons on the right in the footer 

## Fixes
Fixes: #https://warthogs.atlassian.net/browse/WD-4834
